time/src/main/java/net/jqwik/time/internal/properties/arbitraries/valueRanges/ValueRange.java

```diff
@@ -0,0 +1,61 @@
+package net.jqwik.time.internal.properties.arbitraries.valueRanges;
+
+public abstract class ValueRange<T> {
```

Comment:
For me the name is too generic for its current usage. It's only used for "Betweens" so I'd call it like that.